crates/turbopack-node/js/src/ipc/index.ts

```diff
@@ -83,6 +83,9 @@ function createIpc<TIncoming, TOutgoing>(
       }
     });
   });
+  socket.once("close", () => {
+    process.exit(0);
```

Comment:
Add a comment to explain how this can happen, and that this avoids zombie processes.